source/documentors/how-tos/develop_docs_locally.rst

```diff
@@ -0,0 +1,132 @@
+.. _Develop Docs Locally:
+
+Develop Docs Locally
+########################
+
+.. tags:: documentor, how-to
+
+
+If you are familiar with the basics of a local development environment, you can
+set one up to develop documentation locally - that is, on your machine rather
+than waiting for the automated build on GitHub.
+
+Prerequisites
+**************
+
+In this How-To guide, we assume that you have the following:
+
+* A Mac or Linux environment
+* Familiarity with the command line (terminal application), or the desire to learn!
+
+Set Up Your Environment
+*************************
+
+#. On a Mac, first install `homebrew <https://brew.sh/>`_ using the command line ("Terminal" application on a Mac)::
+
+	/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+#. Install Python, using the version specified in `.readthedocs.yaml <https://github.com/openedx/docs.openedx.org/blob/main/.readthedocs.yaml>`_::
+
+	# On Mac
+	brew install python3.12 # Insert proper version here
+
+#. Install Git::
+
+	# On Mac
+	brew install git
+
+#. Create a directory to hold your openedx code (I call it ``openedx`` but you
+   can call it anything) and switch to that directory::
+
+	mkdir openedx
+	cd openedx
+
+#. Create a new Virtualenv for docs development (I call it "vdocs.openedx.org",
+   but you can use any name that makes sense to you)::
+
+	python3.12 -m venv vdocs.openedx.org # Use proper Python version here
```

Comment:
Does this venv name have any significance? I think the convention is usually `venv` or `.venv` if it's in the repo your working with.

Reply:
No, I didn't know there was a convention at all for venvs, I usually make them outside of my git repository with this name 🤷🏻‍♀️ I've been doing it this way for like 20 years

Can you point me at some docs that explain the convention and I'll update this, or you could just suggest what this section should say?